packages/react-core/src/next/components/Wizard/WizardToggle.tsx

```diff
@@ -86,7 +86,7 @@ export const WizardToggle = ({
 
   const wizardNav = isCustomWizardNav(nav)
     ? nav(isNavOpen, steps, activeStep, goToStepByIndex)
-    : React.useMemo(() => {
```

Comment:
Can we keep this memoization and pull this out into a constant above called `defaultWizardNav`, then use that in this ternary?

Reply:
@jeffpuzzo I restored the memoization in a slightly different manner. WDYT?

Reply:
Works for me, thanks!